cpu/saml21/cpu.c

```diff
@@ -23,6 +23,14 @@
 #include "periph_conf.h"
 #include "stdio_base.h"
 
+/* As long as DFLL & DPLL are not used, we can default to
+   always using the buck converter when available. */
+#if !defined(CPU_SAMR30)
+#define USE_VREG_BUCK   (1)
+#else
+#define USE_VREG_BUCK   (0)
+#endif
```

Comment:
USE_VREG_BUCK should be a board parameter.
Some custom boards don't have the required coil to use VBUCK so unfortunately we cannot assign it at CPU level.
This is the same for every SAM0-based boards.

Reply:
Ah just like nrf52 - #13586
Do you know if there is any board supported by RIOT that *doesn't* have the coil? 
But looks like right now there are only `-xpro` boards with those MCUs so they are expected to have all the bells and whistles.

Reply:
@benpicco 
Indeed, all Microchip devboards are equipped with the external coil. But we don't know all the boards in the wild so better take extra care there :)

Reply:
Yes I followed the example of `nrf5x` here.